src/extension/src/utils/vscodeUI.ts

```diff
@@ -30,7 +31,10 @@ export class VSCodeUI extends WizardServant {
   }
 
   async promptUsersToResetPages(message: any) {
-    if (message.payload.pagesLength > 0) {
+    if (
+      message.payload.pagesLength > 0 &&
+      message.text == PAYLOAD_MESSAGES_TEXT.SWITCH_FRAMEWORKS_TEXT
```

Comment:
[It's best to always use === to check for value equality](https://www.impressivewebs.com/why-use-triple-equals-javascipt/)